ChangeLog.md

```diff
@@ -1,5 +1,181 @@
 # Changelog for haskell-language-server
 
+## 2.7.0.0
+
+- Bindists for GHC 9.8.2
```

Comment:
We also enable many more plugins for GHC 9.8.

Reply:
Ah yes, that slipped through due to existing docs updates.